### DISCUSSION TITLE
Add "Front end/customer enabled" status for countries

### DISCUSSION THREAD

**Discussion by ryanmwc** [craftcms/commerce#2422] [Ideas]
**Description**
I have 3 countries enabled in my store settings, which is great for offering a list to choose from when a customer is filling out a shipping address (since we only ship in these 3 countries). However, for billing, we offer every country because the customer can pay from anywhere.  Saving this to the cart works fine, but when I view the saved addresses in the CP, the country dropdown only offers the 3 enabled countries and not every country whether I view it in the cart or in my customer profile.

**Expected behavior**
I expect to be able to choose any country when editing addresses in the CP. Offering enabled countries should be something you do on the frontend.

I feel like this was recently changed, but if not, it would make sense to allow editing the country to be anything in the CP.

**Additional info**
- Craft version: 3.5.11.1
- Commerce version: 3.2.7
- PHP version: 7.3

**Comment by nfourtythree** [comment]
Hi @ryanmwc 

Thanks for your feedback.

I am going to mark this as a feature request. In the future, we will look to add a "Front end enabled" status to countries alongside the current enabled status. This will give finer control over what is displayed (similar to how gateways work).

For the moment my recommendation would be to enable all the countries you require (which will allow selection in the CP) and then update your front end code to only show the countries you allow a customer to select.

Hope this helps, thanks!
